src/bundler/builders/defaultsMode/cjs.js

```diff
@@ -1,29 +1,22 @@
 import packageResult from 'utils/packageResult';
 
 export default function cjs ( bundle, body, options ) {
-	var importBlock,
-		x,
-		intro,
-		indentStr,
-		defaultName;
-
-	indentStr = body.getIndentString();
-
-	importBlock = bundle.externalModules.map( x => {
+	var importBlock = bundle.externalModules.map( x => {
 		var name = bundle.uniqueNames[ x.id ];
-		return indentStr + `var ${name}__default = require('${x.id}');`;
+		return `var ${name}__default = require('${x.id}');`;
 	}).join( '\n' );
 
 	if ( importBlock ) {
 		body.prepend( importBlock + '\n\n' );
 	}
 
-	if ( defaultName = bundle.entryModule.identifierReplacements.default ) {
-		body.append( `\n\n${indentStr}module.exports = ${defaultName};` );
+	var defaultName = bundle.entryModule.identifierReplacements.default;
+	if ( defaultName ) {
+		body.append( `\n\nmodule.exports = ${defaultName};` );
 	}
 
-	intro = '(function () {\n\n' + indentStr + "'use strict';\n\n";
+	body.prepend("'use strict';\n\n");
 
-	body.prepend( intro ).trim().append( '\n\n}).call(global);' );
+	body.indent().prepend('(function () {\n\n').append('\n\n}).call(global);');
```

Comment:
I think we should have `.trimLines()` between the prepend and append, so we don't get excess empty lines at the end, unless there's any reason not to?